drivers/gpio/gpio_emul.c

```diff
@@ -90,8 +90,8 @@ struct gpio_emul_data {
 	gpio_port_value_t output_vals;
 	/** Interrupt status for each pin */
 	gpio_port_pins_t interrupts;
-	/** Mutex to synchronize accesses to driver data and config */
-	struct k_mutex mu;
+	/** Spinlock to synchronize accesses to driver data and config */
```

Comment:
Just a couple of lines above here, can you change `@a mu` to `@a lock`?

Reply:
Sorry - missed that prior to my approval

Reply:
Found some more mu...
Sorry, I'm really bad when it comes to documentation.